package.json

```diff
@@ -102,15 +104,12 @@
     "three": "0.87.0",
     "typeahead.js": "^0.11.1",
     "typedoc": "^0.11.1",
-    "typescript": "^2.0.0",
     "typeface-roboto": "0.0.54",
     "underscore": "^1.8.3",
     "url-join": "4.0.0",
     "url-loader": "^0.5.8",
     "velocity.java": "^1.3.1",
     "webpack": "3.6.0",
-    "webpack-dev-server": "2.8.2",
-    "react-dropzone": "8.0.3"
```

Comment:
@rodriguez-facundo  @afonsobspinto is dropzone what we added for HNN? Should it be moved to HNN if that is the case? If not merging this would break that. cc @filippomc

Reply:
@tarelli maybe I should revert the accidentally removed dependencies here and leave the discussion for a future refactoring?